src/main/java/org/team1540/elmo/subsystems/ejectors/ResetEjectorAndWaitForeverCommand.java

```diff
@@ -1,19 +1,23 @@
 package org.team1540.elmo.subsystems.ejectors;
 
 import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.RunCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import org.team1540.elmo.utils.WaitDashboardVarCommand;
 
-public class ResetEjectorCommand extends SequentialCommandGroup {
-    public ResetEjectorCommand(Ejector ejector) {
+public class ResetEjectorAndWaitForeverCommand extends SequentialCommandGroup {
+    public ResetEjectorAndWaitForeverCommand(Ejector ejector, boolean sucking) {
         addRequirements(ejector);
         addCommands(
                 parallel(
                         new InstantCommand(()->ejector.setExtended(false)),
-                        new InstantCommand(()->ejector.setSuction(false))
-                        // new InstantCommand(()->ejector.setSuction(true))
+                        new InstantCommand(()->ejector.setSuction(sucking))
                 ),
-                new WaitDashboardVarCommand(0.5,"retract",this)
+                new WaitDashboardVarCommand(0.5,"retract",this),
+                new RunCommand(()->{})
```

Comment:
You can also force this to never end by overriding isFinished and never returning true, but this way seems just as good

Reply:
That doesnt work for sequential command groups D;